R/apply_filters.R

```diff
@@ -22,7 +22,7 @@
 #' @param start_analysis Numeric. A vector of length 1 indicating the start
 #'   year of the analysis.
 
-apply_filters <- function(data,
+apply_filters_old <- function(data,
                           investor = NULL,
                           sectors = NULL,
                           technologies = NULL,
```

Comment:
Style please.

Reply:
Does `_old` mean "dead code"? If so, can we delete it?

You can always find an old function, say `apply_filters`, with something like this:

```r
git log -S "apply_filters <- function" -p
```

Reply:
_old means it is used by the web script in the interactive report, because we do not want to interfere with results of past projects.

Reply:
Thanks, so it's definitely not dead code. Then for maximum backward compatibility it seems easier to use `_new()` for new stuff rather than `_old` for old stuff -- that way you don't need to change any existing code.

Reply:
quick general comment: The plan is to delete old webtool functions (i.e. _old) functions as soon as we are allowed to. That is why we prefer to give those ones the clumsy name _old then to use _new on the functions that will soon be the only ones.